src/libstd/sys/wasm/process.rs

```diff
@@ -38,19 +40,16 @@ pub enum Stdio {
 
 impl Command {
     pub fn new(_program: &OsStr) -> Command {
-        Command {}
+        Command {
+            env: Default::default()
+        }
     }
 
     pub fn arg(&mut self, _arg: &OsStr) {
     }
 
-    pub fn env(&mut self, _key: &OsStr, _val: &OsStr) {
-    }
-
-    pub fn env_remove(&mut self, _key: &OsStr) {
-    }
-
-    pub fn env_clear(&mut self) {
+    pub fn env_mut(&mut self) -> CommandEnv<DefaultEnvKey> {
```

Comment:
Moar wasm typo (missing `&mut`).

```
[00:39:21] error[E0308]: mismatched types
[00:39:21]   --> /checkout/src/libstd/sys/wasm/process.rs:52:9
[00:39:21]    |
[00:39:21] 51 |     pub fn env_mut(&mut self) -> CommandEnv<DefaultEnvKey> {
[00:39:21]    |                                  ------------------------- expected `sys_common::process::CommandEnv<sys_common::process::DefaultEnvKey>` because of return type
[00:39:21] 52 |         &mut self.env
[00:39:21]    |         ^^^^^^^^^^^^^ expected struct `sys_common::process::CommandEnv`, found mutable reference
[00:39:21]    |
[00:39:21]    = note: expected type `sys_common::process::CommandEnv<sys_common::process::DefaultEnvKey>`
[00:39:21]               found type `&mut sys_common::process::CommandEnv<sys_common::process::DefaultEnvKey>`
```